corehq/apps/builds/management/commands/commcare_build_importer.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import requests
+
+from cStringIO import StringIO
+from jenkinsapi.jenkins import Jenkins
+
+from corehq.apps.builds.models import CommCareBuild
+from django.core.management.base import BaseCommand, CommandError
+
+
+# supress requests library INFO logs
+logging.getLogger("requests").setLevel(logging.WARNING)
+
+
+class Command(BaseCommand):
+    """
+    This is an automated version of -
+    https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/builds#adding-commcare-j2me-builds-to-commcare-hq
+    """
+    help = "Interactively import a build from Jenkins build server."
+
+    def handle(self, *args, **options):
+        if not args:
+            self.interactive_import()
+        else:
+            raise CommandError(self.help)
+
+    def interactive_import(self):
+        print "Pinging Jenkins build server. Pelase wait..."
+        build_server = Jenkins("http://jenkins.dimagi.com")
+
+        # let user choose one of ['commcare-mobile-2.21', 'commcare-mobile-2.22']
+        jenkin_projects = build_server.keys()
+        selected_project_key = None
+        print "Jenkins has following projects. Please choose one"
+        print [p for p in jenkin_projects if 'commcare-mobile' in p]
+        while selected_project_key not in jenkin_projects:
+            selected_project_key = raw_input("")
```

Comment:
What if the user decides she wants to cancel? Might want to provide an option for that (empty string?).